mailbox/api/src/main/java/org/apache/james/mailbox/events/EventBus.java

```diff
@@ -26,6 +26,9 @@
 import reactor.core.publisher.Mono;
 
 public interface EventBus {
+
+    interface MagicEventBus extends EventBus, UninitializedEventBus {}
```

Comment:
Should we really call something `Magic`?

Reply:
No.

Read the PR description.